Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -261,14 +272,13 @@ namespace AZ
             // End of functions accessed by Scene class
             //////////////////////////////////////////////////
 
-
             RenderMode m_renderMode = RenderMode::RenderEveryTick;
 
             // The Scene this pipeline was added to.
             Scene* m_scene = nullptr;
 
-            // Pass tree which contains all the passes in this render pipeline.
-            Ptr<ParentPass> m_rootPass;
+            // Holds the passes belonging to the pipeline
+            PassContainer m_passes;
```

Comment:
Nit naming: m_passes -> m_passTree

Reply:
Hmm, so I would update the PassContainer to be called PassTree too? Could work.
Technically any parent pass is itself a pass tree, though probably not a big deal sure people could still understand PassTree.